src/config.ts

```diff
@@ -20,6 +20,22 @@ export default {
         rust: 'rustc',
         java: 'javac',
     },
+    compilerToId: {
+        'GNU G++17 7.3.0': 54,
+        'GNU G++14 6.4.0': 50,
+        'GNU G++11 5.1.0': 42,
+        'Microsoft Visual C++ 2017': 59,
+        'Microsoft Visual C++ 2010': 2,
+        'Clang++17 Diagnostics': 52,
+        'Java 11.0.5': 60,
+        'Java 1.8.0_162': 36,
+        'PyPy 3.6 (7.2.0)': 41,
+        'Python 3.7.2': 31,
+        'PyPy 2.7 (7.2.0)': 40,
+        'Python 2.7.15': 7,
+        'GNU GCC C11 5.1.0': 43,
+        'Rust 1.35.0': 49,
+    },
```

Comment:
Is this the right place to keep this? Please let me know if this needs to be shifted somewhere else.

Reply:
Yeah, this looks like a good place.